src/App.js

```diff
@@ -50,10 +50,7 @@ const App = ({ setClient }) => {
 
   useNetService(client)
 
-  const { initialScreen, initialRoute, isLoading } = useAppBootstrap(
-    client,
-    setClient
-  )
+  const { initialRoute, isLoading } = useAppBootstrap(client)
```

Comment:
maybe we could use `initialParams` instead of `initialRoute.params`? I think that would go well with the spirit of this PR

Reply:
`initialRoute` and `initialParams` are part of the same concept (i.e. deep links) so I would prefer to have them "linked" into the same `useState` (even if today react is able to handle multiple setState in a single re-render)

However I can split them in the hook's return. Something like:
```js
  return {
    client,
    initialRoute: initialRoute.route,
    initialParams: initialRoute.params
    isLoading
  }
```
But I'm not sure this will improve clarity that much

Reply:
alright agree with you